### ISSUE TITLE
Dark mode for Typst Documentation

### ISSUE THREAD

**Issue by kamack38** [typst/webapp-issues#207]
### Description

It would be great and logical to have a dark mode for the documentation page, since the editor has. One of the challenges would be to generate previews with a black or gray background and white text.

### Use Case

It would reduce eye strain, especially when switching from the black editor to the docs page.

**Comment by Andrew15-5** [comment]
I agree that images should be left alone, because that's how dark reader works. And it adds complexity if 2 different images should be generated. Although I wish all images appear dark-friendly.

**Comment by Andrew15-5** [comment]
@kamack38, this screenshot appears outdated. I've added one or more patches to dark mode so that the docs would look decent. I remember that at least I patched the SVG images. So now all the arrows and icons are visible.

I honestly don't care if any website has a dark theme as long as it looks readable and not ugly with dark reader.
